File: C/numbers/Palindrome.c
#include <stdio.h>
int main()
{
    int num, rev = 0, d, temp;
    printf("Enter a number : ");
    scanf("%d", &num);
    temp = num;
    while (num > 0)
    {
        d = num % 10;
        num = num / 10;
        rev = rev * 10 + d;
    }
    if (temp == rev)
    {
        printf("The number is palindrome : %d = %d", temp, rev);
    }
    else
        printf("Not a palindrome number");
    return 0;
}

File: python/basics/palindrome.py
#Write a program to check a given number is palindrome or not, using while or for loop
# num=int(input("Enter a number : "))
# temp=num
# rev=0
# while temp>0:
#     d=temp%10
#     temp=temp//10
#     rev=rev*10+d
# if (rev==num):
#     print("The number is palindrome")
# else:
#     print("The number is not palindrome")

#Write a really simple program to check a given number is palindrome or not, using for loop
num= (input("Enter a number : "))
rev=""
for digit in num :
    rev = digit + rev
if num == rev :
    print("The number is palindrome")
else:    
    print("The number is not palindrome")

